src/Gridify.Elasticsearch/GridifyExtensions.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Elastic.Clients.Elasticsearch;
+using Elastic.Clients.Elasticsearch.QueryDsl;
+using Gridify.Syntax;
+
+namespace Gridify.Elasticsearch;
+
+public static class GridifyExtensions
+{
+   public static Query ToElasticsearchQuery<T>(this string? filter, IGridifyMapper<T>? mapper = null)
```

Comment:
it is okay to have these extension methods but to keep the constancy in the library, I would prefer to have Similar extension methods like `ApplyFiltering` `ApplyOrdering` on the `Query` object instead. unless you have a specific reason for this design choice?